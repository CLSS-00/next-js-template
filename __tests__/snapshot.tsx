import React from 'react'
import renderer from 'react-test-renderer'
import Index from '../pages/index'

it('renders homepage unchanged', () => {
  const { act } = renderer
  let tree
  act(() => {
    tree = renderer.create(<Index />)
  })

  expect(tree).toMatchSnapshot()
})
