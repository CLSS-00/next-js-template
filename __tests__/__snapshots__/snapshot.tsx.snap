// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div
  className="container"
>
  <main
    className="main"
  >
    <h1
      className="title"
    >
      Welcome to 
      <a
        href="https://nextjs.org"
      >
        Next.js!
      </a>
    </h1>
  </main>
  <footer
    className="footer"
  >
    <a
      href=""
      rel="noopener noreferrer"
      target="_blank"
    >
      Powered by 
      <span
        className="logo"
      />
    </a>
  </footer>
</div>
`;
